src/backend/migrations/cassandra/Cassandra.h

```diff
@@ -0,0 +1,23 @@
+#include <backend/CassandraBackend.h>
+#include <backend/migrations/Migrations.h>
+
+namespace Backend {
+namespace Migrations {
+namespace Cassandra {
+
+// IMPORTANT! The names of each migration function cannot change. If they do, it
```

Comment:
That's also one reason to use version number instead and have `static constexpr uint32_t version = 2;` or something like this here and a corresponding value in the database (representing the actual current schema).